src/Memory/ILogStorage.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import * as CLM from '@conversationlearner/models'
+
+export interface LogQueryResult {
+    logDialogs: CLM.LogDialog[]
+    continuationToken: string | undefined
+}
+
+export interface ILogStorage {
+
+    /** Add a log dialog to storage */
+    Add(appId: string, logDialog: CLM.LogDialog): Promise<CLM.LogDialog>
+
+    /** Retrieve a log dialog from storage */
+    Get(appId: string, logDialogId: string): Promise<CLM.LogDialog | undefined>
```

Comment:
Since this is new and isn't bound by legacy reasons would use the `modelId` instead of `appId`

Reply:
Feels confusing to me to use appId in some places and modelId in others.